pkg/vtpm/Dockerfile

```diff
@@ -26,31 +27,38 @@ RUN ./configure --datarootdir=/usr/share/ && \
     make -j "$(getconf _NPROCESSORS_ONLN)" && make install
 
 WORKDIR /tpm2-tss
-ADD https://github.com/tpm2-software/tpm2-tss/tarball/2.3.1 /tpm2-tss.tgz
+ADD https://github.com/tpm2-software/tpm2-tss/tarball/4.0.1 /tpm2-tss.tgz
 RUN tar -zxvf /tpm2-tss.tgz --strip-components=1 && \
     rm /tpm2-tss.tgz
+
+# because various builds assume this is in a git repo
```

Comment:
Instead of doing this you can just clone directly the git repo using `--keep-git-dir=true`, like the following:

`ADD --keep-git-dir=true https://github.com/tpm2-software/tpm2-tss.git#4.0.1`